config.json

```diff
@@ -61,6 +61,11 @@
     "show_accept_visa": false,
     "product_list_display_mode": "grid",
     "logo-position": "center",
+    "logoSize": "250x100",
```

Comment:
We are using snake_case for all our settings, can you change that to keep consistency?